test/kitchen/site-cookbooks/dd-agent-install-script/recipes/default.rb

```diff
@@ -50,7 +50,7 @@
   cwd wrk_dir
   command <<-EOF
     sed -i '1iDD_API_KEY=#{node['dd-agent-install-script']['api_key']}' install-script
-    sed -i '1iDD_URL="datad0g.com"' install-script
+    sed -i '1iREPO_URL="datad0g.com"' install-script
```

Comment:
So I noticed something for all these `sed` commands. We have a few lines at the top of the script, most importantly we have the script shebang at the top `#!/bin/bash`, so we don't want to put anything above that. We then have a few lines with the header I  think 7 lines in total.

To me, this should be at the very least:
```
sed -i '1aREPO_URL="datad0g.com"' install-script
```
to avoid putting anything over the shebang.

Also, we probably want to set the `DD_URL` as well, no?